### DISCUSSION TITLE
Export basic states

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#693] [Ideas]
I think kaboom game states like the current scene, the options selected in `kaboom()` like `crisp` and `pixelDensity`, the current playing sounds, etc.

**Comment by lajbel** [comment]
Seeing this is related, maybe a method like `getAsset` would be good. We can load assets using `load()` in the `custom` internal assets, but we can't get it.

**Reply by slmjkdbtl** [reply]
You can use `getSprite()` etc

**Reply by lajbel** [reply]
I have a plugin to load jsons inside assets, there's a way to get it back?

**Reply by slmjkdbtl** [reply]
Oh you mean specifically getting custom stuff loaded by `load()`?

**Reply by lajbel** [reply]
yep!

**Comment by slmjkdbtl** [comment]
Good idea, will export these
